reco_utils/recommender/deeprec/models/base_model.py

```diff
@@ -406,9 +406,10 @@ def fit(self, train_file, valid_file, test_file=None):
 
             if self.hparams.save_model:
                 if epoch % self.hparams.save_epoch == 0:
+                    save_path_str = self.hparams.MODEL_DIR + "/" + "epoch_" + str(epoch)
```

Comment:
please change to:
```
save_path_str = os.path.join(self.hparams.MODEL_DIR, "epoch_" + str(epoch))
```
that way it can be used in all OS

Reply:
Hello @miguelfierro,

Sure thing - will make the change and get back to you.